transports/msmq/index.md

```diff
@@ -77,36 +77,36 @@ Although MSMQ has the concept of both [Public and Private queues](https://techne
 
-When an endpoint sends a command or event message that source requires permission at the target queue. For endpoints using the either the publish-subscribe or request-response patterns, both parties actually send messages and must have permission to send messages to each other. This is to allow the Subscribe message to be sent from the subscribing endpoint to the publishing endpoint in case of publish-subscribe or the response to be send back to the client in case of request-response.
+Any endpoint that sends a message to a target endpoint requires the `Send` permission to be granted for the sending user account on the target queue. For example, if an `endpoint A` is running as `userA` and is sending a message to `endpoint B`, then `userA` requires the `Send` permission to be granted on `endpoint B`'s queue. When using messaging patterns like request-response or publish-subscribe, the queues for both the endpoints will require `Send` permissions to be granted to each others user accounts. 
 
-NOTE: In versions 5.2.20 and 6.0.3 a bug was fixed where installers always changed permissions on a queue, also if it already existed.
-
-When an endpoint creates a queue on a machine, the default permissions depend on whether the machine is connected to a domain or a workgroup.
+When an endpoint creates a queue on a machine, the default permissions depend on whether the server is joined to a [domain or a workgroup](https://support.microsoft.com/en-us/help/884974/information-about-workgroup-mode-and-about-domain-mode-in-microsoft-me).
 
 
 ### Domain mode
 
-If the machine is connected to a domain, then only the domain user running the endpoint when the queue is created will have Send permissions granted. If multiple endpoints need to communicate and are running under the same domain account, no further configuration is required. If multiple endpoints need to communicate using different domain accounts then the Send permission on the receiving endpoint input queue needs to be granted to the domain account of the sending endpoint.
+If the machine is joined to a domain, then at the time of queue creation, only the domain user that created the queue, domain administrators and local administrators will have `Send` permissions granted. The `Everyone` user group and `Anonymous` user group will not have `Send` permissions. If all the endpoints which need to communicate are running under the same domain account, no further configuration is required. However, if the endpoints are run using different domain accounts, then the `Send` permission on the receiving endpoint's input queue needs to be explicitly granted to the domain user account of the sending endpoint.
```

Comment:
No, because that's also default windows behavior when joined to a domain.  However I do see your point that we explicitly add administrators permissions. I think its safe to not specify either. As we have clearly stated the correct behavior.